### DISCUSSION TITLE
option --auto-reconnect=5m (5 menit)

### DISCUSSION THREAD

**Discussion by laksa19** [aldinokemal/go-whatsapp-web-multidevice#196] [Ideas]
terima kasih sebelumnya ini sangat membantu.

bisa tidak ditambahkan option --auto-reconnect agar tidak perlu klik manual, entah kenapa akhir2 ini sering logout sendiri.

terima kasih

**Comment by aldinokemal** [comment]
menarik, sepertinya bisa saya masukkan ke mandatory fitur tanpa harus menambahkan --auto-reconnect

**Reply by aldinokemal** [reply]
@laksa19 bisa dicoba pake versi [4.19](https://github.com/aldinokemal/go-whatsapp-web-multidevice/releases/tag/v4.19.0)

**Reply by laksa19** [reply]
siap, terima kasih
